package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A Node.js starter for the Serverless Framework with async/await and unit test support",
   "main": "handler.js",
   "scripts": {
-    "test": "serverless-bundle test"
+    "test": "serverless-bundle test",
+    "lint": "serverless-bundle eslint"
```

Comment:
Might be better to move eslint before the test,
Just so you get earlier feedback, as the tests takes a few mins to run

Reply:
npm doesnt run the scripts in the order they are defined.